djangular/forms/angular_model.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, data=None, *args, **kwargs):
             self.ng_directives['ng-model'] = '%(model)s'
         self.prefix = kwargs.get('prefix')
         if self.prefix and data:
-            data = dict((self.add_prefix(name), value) for name, value in data.get(self.prefix).items())
+            data = {name: value for name, value in data.items() if name.startswith(self.prefix + '.')}
```

Comment:
AFAIK, this syntax for comprehensions does not work for Py2.
Does this work on python-2.7?